docs/opencost-exporter.md

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 5
+---
+# OpenCost exporter
+
+Running OpenCost as a Prometheus metric exporter allows you to export various cost metrics to Prometheus without setting up any other OpenCost dependencies. Doing so lets you write PromQL queries to calculate the cost and efficiency of any Kubernetes concept, e.g. namespace, service, label, deployment, etc. You can also calculate the cost of different Kubernetes resources, e.g. nodes, PVs, LoadBalancers, and more. Finally, you can do other interesting things like create custom alerts via AlertManager and custom dashboards via Grafana.
+
+## Installing
+
+> Note: all deployments of Kubecost function as a Prometheus metric exporter. We strongly recommend helm as an install path to take advantage of Kubecost’s full potential. [View recommended install](http://docs.kubecost.com/install).
+
+If you would prefer to not use the recommended install option and just deploy the OpenCost open source cost model as a metric exporter, you can follow these steps:
+
+1. This will target a Prometheus Server named my-prometheus-server in the prom namesapace. If yours is somewhere else, you will need to modify accordingly. See `Prerequisites` in [basic-setup.md](./basic-setup.md)
+
+2. Apply the combined YAML:
+
+    ```
+    kubectl apply -n opencost -f https://raw.githubusercontent.com/kubecost/opencost/develop/kubernetes/exporter/exporter.yaml
+    ```
+
+3. To verify that metrics are available:
+
+    ```
+    kubectl port-forward --namespace opencost service/opencost 9003
+    ```
+
+    Visit [http://localhost:9003/metrics](http://localhost:9003/metrics) to see exported metrics
+
+Add Kubecost scrape config to Prometheus ([more info](https://prometheus.io/docs/introduction/first_steps/#configuring-prometheus))
```

Comment:
Can we rewrite this line? Maybe something like "More info for adding Kubecost scrape config to Prometheus [here]."